### DISCUSSION TITLE
MQTT or POST

### DISCUSSION THREAD

**Discussion by MassakerGU** [Drumber/RemoteLight#14] [Ideas]
Is something planned to change the effect of the stripes over MQTT or a http POST. 
That would be a really nice feature.

**Comment by Drumber** [comment]
Hey, I actually started working on a REST API just 2 days ago. So it will be possible to control effects, outputs and more using HTTP requests.
You can see the progress on the [rest_api branch](https://github.com/Drumber/RemoteLight/tree/rest_api/remotelight-restapi).

**Reply by MassakerGU** [reply]
Wow that is really cool. I dream to control all my LED in my Pc with Remotelight/OpenRGB over Home Assistant. Really cool project.

**Reply by Drumber** [reply]
Any comments or suggestions are welcome!
